services/web-app/pages/designing/figma.mdx

```diff
@@ -66,14 +72,15 @@ on (D).
 <br />
 <br />
 
-To preview the four Carbon themes, visit these view-only links.
+To preview the four v11 Carbon theme libraries, visit the following view-only links. To access the
+v10 Figma libraries, see [v10 Design Kits](https://v10.carbondesignsystem.com/designing/kits/figma).
```

Comment:
This link is going the the v10 website, is that what we want to redirect to in Next? Or should we link to somewhere in platform?

Reply:
Good point. The only way we could direct users to this on the platform currently is to send them to the design kit catalog with the figma filter, though we can't filter by v10 right now. We have another issue to add version numbers to all library names. Does that work?

Reply:
Yeah I think that would work with a little rephasing of the call to action.

Reply:
Cool. Added that link and revised the cta